src/bwc_erc20_token.cairo

```diff
@@ -1,16 +1,215 @@
+use starknet::ContractAddress;
+#[starknet::interface]
+trait IERC20<TContractState> {
+    fn get_name(self: @TContractState) -> felt252;
+    fn get_symbol(self: @TContractState) -> felt252;
+    fn get_decimals(self: @TContractState) -> u8;
+    fn get_total_supply(self: @TContractState) -> u256;
+    fn balance_of(self: @TContractState, account: ContractAddress) -> u256;
+    fn allowance(self: @TContractState, owner: ContractAddress, spender: ContractAddress) -> u256;
+    fn transfer(ref self: TContractState, recipient: ContractAddress, amount: u256);
+    fn transfer_from(
+        ref self: TContractState, sender: ContractAddress, recipient: ContractAddress, amount: u256
+    );
+    fn approve(ref self: TContractState, spender: ContractAddress, amount: u256);
+    fn increase_allowance(ref self: TContractState, spender: ContractAddress, added_value: u256);
+    fn decrease_allowance(
+        ref self: TContractState, spender: ContractAddress, subtracted_value: u256
+    );
+}
+
 #[starknet::contract]
+mod ERC20 {
+    // importing necessary libraries
+    use starknet::ContractAddress;
+    use starknet::get_caller_address;
+    use starknet::contract_address_const; //similar to address(0) in Solidity
+    use zeroable::Zeroable;
```

Comment:
this contract is not compiling due to `use zeroable::Zeroable;`

import appropriately using: 
`use core::zeroable::Zeroable;`